### DISCUSSION TITLE
Possible Improvment: $(SolutionDir)

### DISCUSSION THREAD

**Discussion by r2d2Proton** [NVIDIA-Omniverse/PhysX#154] [Ideas]
I wish I knew CMake better, it does look like a cool tool. As you may know Visual Studio does have a few macros that may be helpful and accessible to CMake. The first image shows the PhysX project configuration properties for a debug build. The second image shows all the projects selected and for all configurations. The end goal is to be able to move projects to different drives or rename top level directories without having dependencies. Please let me know if you have any interest in discussing this or me submitting changes (but would need some help with CMake changes).

![PhysXDebugProps](https://user-images.githubusercontent.com/4054178/218130533-d400a168-2efa-4ce6-9b31-fef3a55dd049.png)

![PhysXAll](https://user-images.githubusercontent.com/4054178/218130650-9cc42eb9-ed00-4cfd-9ff9-4db76b8ec346.png)

**Comment by ayoub-belarbi** [comment]
Hello @r2d2Proton,
Thank you for your proposition. I would like to convert this into a conversation if you don't mind. Since it's not an issue. We can discuss there how to implement this in cmake.
Thanks
